medusa/providers/torrent/json/btn.py

```diff
@@ -262,10 +262,13 @@ def _api_call(self, apikey, params=None, results_per_page=300, offset=0):
             else:
                 logger.log('JSON-RPC protocol error while accessing provider. Error: {msg!r}'.format
                            (msg=error.message[1]), logger.ERROR)
-
+        
         except (socket.error, socket.timeout, ValueError) as error:
             logger.log('Error while accessing provider. Error: {msg}'.format
                        (msg=error), logger.WARNING)
+        except Exception as error:
+            logger.log('Unknown error while accessing provider. Error: {msg}'.format
+                      (msg=error), logger.ERROR)
```

Comment:
@duramato don't need this. rato's global exception already catch unexpected erros